### PR TITLE
[DO NOT MERGE] Update all non-workshop images to latest

### PR DESCRIPTION
**Pull request by jsignell**
This is a test of the new images, this PR updates all images to new names and latest version. 

First batch of integration tests running at https://github.com/saturncloud/release-images/runs/7638367004?check_suite_focus=true

### CODE REVIEW

**Comment by jsignell** [comment]
Take 2 with integration test changes in place: https://github.com/saturncloud/release-images/runs/7653199285?check_suite_focus=true

File: examples/snowflake/.saturn/saturn.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "example-snowflake",
-  "image_uri": "public.ecr.aws/saturncloud/saturn-rapids:2022.03.01",
+  "image_uri": "public.ecr.aws/saturncloud/saturn-r:2022.06.01",
```
Comment (jsignell) [inline examples/snowflake/.saturn/saturn.json]:
```suggestion
  "image_uri": "public.ecr.aws/saturncloud/saturn-python-rapids:2022.06.01",
```

**Comment by jsignell** [comment]
There are 2 remaining issue:
- r-tensorflow, seems like the `RETICULATE_PYTHON` is pointing to the wrong spot
- python-rapids: `TypeError: C function cuda.ccudart.cudaStreamSynchronize has wrong signature (expected __pyx_t_4cuda_7ccudart_cudaError_t (__pyx_t_4cuda_7ccudart_cudaStream_t), got cudaError_t (cudaStream_t))`

**Comment by hhuuggoo** [comment]
@jsignell I'm merging this. CI is failing on rf-rapids-dask.ipynb from snowflake, but when I run it, I can't reproduce the failure. I'm merging this so I can get everything on to internal via the template, and then I can track down what's going on. (I'll fix linting first)

**Pull request merged**